subworkflows/local/snpeff.nf

```diff
@@ -21,6 +21,7 @@ workflow SNPEFF {
 
     SNPEFF_ANN (
         vcf,
+        params.snpeffconfig,
```

Comment:
`params.snpeffconfig` may need to be interpolated here with ${ }, but I'm not 100% sure.  If this has already been tested and confirmed working, then ignore this comment.

Reply:
Yes, this has been tested and confirmed its working